azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/utilities/BottomCellActionViewHolder.kt

```diff
@@ -0,0 +1,91 @@
+package com.azure.android.communication.ui.utilities
+
+import android.view.View
+import android.widget.ImageView
+import androidx.core.content.ContextCompat
+import androidx.core.view.AccessibilityDelegateCompat
+import androidx.core.view.ViewCompat
+import androidx.core.view.accessibility.AccessibilityNodeInfoCompat
+import com.azure.android.communication.ui.R
+import com.microsoft.fluentui.persona.AvatarView
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+internal class BottomCellActionViewHolder(itemView: View) : BottomCellViewHolder(itemView) {
+    private val imageView: ImageView = itemView.findViewById(R.id.cell_icon)
+    private val avatarView: AvatarView =
+        itemView.findViewById(R.id.azure_communication_ui_participant_list_avatar)
+    private val accessoryImage: ImageView = itemView.findViewById(R.id.cell_check_mark)
+    private var onClickAction: Runnable? = null
+    init {
+        itemView.setOnClickListener {
+            onClickAction?.run()
+        }
+    }
+
+    override fun setCellData(bottomCellItem: BottomCellItem) {
+        super.setCellData(bottomCellItem)
+        if (bottomCellItem.itemType == BottomCellItemType.BottomMenuTitle) {
+            // Exit with early return because setting the title is the only thing we need to do
+            return
+        }
+        // force bitmap update be setting resource to 0
+        avatarView.setImageResource(0)
+        if (bottomCellItem.icon == null) {
+            ViewCompat.setAccessibilityDelegate(
+                itemView,
+                object : AccessibilityDelegateCompat() {
+                    override fun onInitializeAccessibilityNodeInfo(
+                        host: View,
+                        info: AccessibilityNodeInfoCompat,
+                    ) {
+                        super.onInitializeAccessibilityNodeInfo(host, info)
+                        info.removeAction(AccessibilityNodeInfoCompat.AccessibilityActionCompat.ACTION_CLICK)
+                        info.isClickable = false
+                    }
+                }
+            )
+            imageView.visibility = View.GONE
+            avatarView.visibility = View.VISIBLE
+            avatarView.name = bottomCellItem.title ?: ""
+            bottomCellItem.personaData?.let { personaData ->
+                personaData.avatarBitmap?.let {
+                    avatarView.avatarImageBitmap = it
+                    avatarView.adjustViewBounds = true
+                    avatarView.scaleType = personaData.scaleType
+                }
+            }
+        } else {
+            imageView.setImageDrawable(bottomCellItem.icon)
+            avatarView.visibility = View.GONE
+        }
+        if (bottomCellItem.accessoryImage != null) {
+            accessoryImage.setImageDrawable(bottomCellItem.accessoryImage)
```

Comment:
Please suffix "View" onto views. We have 2 "accessoryImage" fields here, where one is A Image and one is a view. 

Should be 

accessoryImage
accessoryImageView

Reply:
I fixed this with 8554bb9 but I don't think I know a way to avoid the not-null check from the previous comment. I'm assuming that this comment supersedes the previous one, please let me know if there's something better I can do there.